app/validate_txs.go

```diff
@@ -34,7 +36,20 @@ func separateTxs(_ client.TxConfig, rawTxs [][]byte) ([][]byte, []*tx.BlobTx) {
 //
 // Side-effect: arranges all normal transactions before all blob transactions.
 func FilterTxs(logger log.Logger, ctx sdk.Context, handler sdk.AnteHandler, txConfig client.TxConfig, txs [][]byte) [][]byte {
-	normalTxs, blobTxs := separateTxs(txConfig, txs)
+	// all transactions should be below the max tx size
+	maxTxSize := appconsts.MaxTxSize(ctx.BlockHeader().Version.App)
+	//nolint:prealloc
+	var txsBelowLimit [][]byte
+	for idx, tx := range txs {
+		if len(tx) > maxTxSize {
+			err := fmt.Sprintf("tx size %d bytes at index %d exceeds the application's configured threshold of %d bytes", len(tx), idx, maxTxSize)
```

Comment:
[uber nit] in some places we call the max tx size a "limit". In other places we call it a "threshold". It would be nice if we always referred to this with the same vocabulary. 

In my opinion it would be sufficient to call it `MaxTxSize` and not use "limit" or "threshold" because the prefix "max" already conveys that.

Reply:
https://github.com/celestiaorg/celestia-app/pull/4084/commits/5cddecc314196fe8e5e1e444c687d16ee502ce56